autoload/dirvish.vim

```diff
@@ -56,6 +56,10 @@ endf
 func! s:fix_dir(dir, silent) abort
   let dir = s:sl(a:dir)
   if !isdirectory(dir)
+    " UNC-style: top-level dir may not exist, while sub dir exists.
```

Comment:
A more concrete example: path `\\wsl$\fedora\` exists, but "parent dir" of it: `\\wsl$\` is not a dir.